tsconfig.esm.json

```diff
@@ -0,0 +1,24 @@
+{
+  "compilerOptions": {
+    "outDir": "./lib/esm",
+    "target": "ESNext",
+    "module": "ESNext",
+    "lib": ["es2020", "esnext"],
+    "typeRoots": ["./node_modules/@types"],
+    "moduleResolution": "node",
+    "noUnusedLocals": true,
+    "noUnusedParameters": true,
+    "noImplicitThis": true,
+    "noImplicitReturns": true,
+    "noImplicitAny": true,
+    "noEmitOnError": true,
+    "noErrorTruncation": true,
+    "skipLibCheck": true,
+    "strict": true,
```

Comment:
It will increase maintainability if we extend from the `tsconfig.json`
https://www.typescriptlang.org/tsconfig#extends